EmailNotifications/Email.cs

```diff
@@ -88,11 +97,16 @@ public void sendEmail()
                 // SEND IN HTML FORMAT (comment this line to send plain text).
                 message.IsBodyHtml = true ;
                 message.Body = outS;
+                string current =  Directory.GetCurrentDirectory();
+                string sFile = current + @"\logo.PNG";
+                Attachment data = new Attachment(sFile, System.Net.Mime.MediaTypeNames.Application.Octet);
+                data.ContentId = "testItem";
+                message.Attachments.Add(data);
                 client.Send(message);
 
     
 
-                // TODO: Replace with the name of your remote SMTP server.
+                
```

Comment:
remove whitespace, I know this has been said but get a formatting tool, will save you all of these nit picking comments